tests/pages/services/TasksTable-cy.js

```diff
@@ -87,4 +87,67 @@ describe("Tasks Table", function() {
       });
     });
   });
+
+  context("Service tasks checkbox", function() {
+    beforeEach(function() {
+      cy.configureCluster({
+        mesos: "1-service-with-executor-task"
+      });
+      cy.visitUrl({ url: "/services/detail/%2Fcassandra/tasks?_k=rh67gf" });
+      cy.get("table tr").find(".form-element-checkbox").as("checkboxes");
+    });
+
+    function assertCheckboxLength() {
+      cy.get("@checkboxes").should("have.length", 2);
+    }
+
+    function assertActionButtons() {
+      cy
+        .get(".filter-bar .button-collection .button-link > span")
+        .eq(0)
+        .contains("Restart");
+      cy.get(".button-collection .button-link > span").eq(1).contains("Stop");
+    }
+
+    it("Select all tasks available and confirm action buttons exist", function() {
+      assertCheckboxLength();
+      cy.get("@checkboxes").eq(0).click();
+      cy
+        .wait(3000)
```

Comment:
I suggested to add this wait, Mat suggested we don't because it's kind of arbitrary. I think I agree with Mat, it sort of clutters the test and might be confusing in the future. You think it's a good idea to have the wait?